### ISSUE TITLE
Wandb.ai does not show any weights or gradients when wandb.watch called on torch.nn.DataParallel instance 

### ISSUE THREAD

**Issue by vadimkantorov** [wandb/client#1819]
I'm using adapted https://github.com/ajabri/videowalk codebase. My wandb.ai dashboard shows logged metrics, but not weights / gradients, despite `wandb.watch(model)` was called (though on `torch.nn.DataParallel` wrapper):
![image](https://user-images.githubusercontent.com/1041752/107234912-72a95900-6a24-11eb-92d4-1f556a53827d.png)

I tried to log in to community slack, but the invite from my wandb account does not work:

![image](https://user-images.githubusercontent.com/1041752/107234449-f7e03e00-6a23-11eb-9db2-97125c88d6d9.png)

**Comment by vadimkantorov** [comment]
When I call `wandb.watch(model_before_dp)`, the Gradients pane is still empty, the model structure is also whown incorrectly

**Comment by cvphelps** [comment]
Hi! It looks like you've only logged 5 steps. You can pass `log_freq=1` to watch if you're only logging 5 steps. By default this is `1000` because these histograms are very weighty, and will very much slow down performance for model training (because computing gradients is expensive).

**Comment by cvphelps** [comment]
For the wandb forum, I'm sorry to hear that the link isn't working! Here's a new link for you: http://wandb.me/slack

**Comment by lavanyashukla** [comment]
@vadimkantorov sorry your slack link didn't work. Where did you get the link from?

**Comment by vadimkantorov** [comment]
Maybe the link at GH Readme: `If you have any questions, please don't hesitate to ask in our Slack community.`

I don't remember anymore :(

**Comment by vadimkantorov** [comment]
> Hi! It looks like you've only logged 5 steps. You can pass `log_freq=1` to watch if you're only logging 5 steps. By default this is `1000` because these histograms are very weighty, and will very much slow down performance for model training (because computing gradients is expensive).

Thanks! I'll check smaller `log_freq`. But if it indeed support DataParallel, it would be good to have an official example

**Comment by sydholl** [comment]
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.
